dataprofiler/tests/profilers/test_data_labeler_column_profile.py

```diff
@@ -391,3 +395,172 @@ def test_empty_data(self, *mocks):
+                            "EMAIL_ADDRESS",
+                            "UUID",
+                            "HASH_OR_KEY",
+                            "IPV4",
+                            "IPV6",
+                            "MAC_ADDRESS",
+                            "PERSON",
+                            "PHONE_NUMBER",
+                            "SSN",
+                            "URL",
+                            "US_STATE",
+                            "INTEGER",
+                            "FLOAT",
+                            "QUANTITY",
+                            "ORDINAL",
+                        ],
+                        "_rank_distribution": {
+                            "UNKNOWN": 0,
+                            "ADDRESS": 0,
+                            "BAN": 0,
+                            "CREDIT_CARD": 0,
+                            "DATE": 0,
+                            "TIME": 0,
+                            "DATETIME": 0,
+                            "DRIVERS_LICENSE": 0,
+                            "EMAIL_ADDRESS": 0,
+                            "UUID": 0,
+                            "HASH_OR_KEY": 0,
+                            "IPV4": 0,
+                            "IPV6": 0,
+                            "MAC_ADDRESS": 0,
+                            "PERSON": 0,
+                            "PHONE_NUMBER": 0,
+                            "SSN": 0,
+                            "URL": 0,
+                            "US_STATE": 0,
+                            "INTEGER": 2,
+                            "FLOAT": 0,
+                            "QUANTITY": 0,
+                            "ORDINAL": 1,
+                        },
+                        "_sum_predictions": [
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            0.0,
+                            2.0,
+                            0.0,
+                            0.0,
+                            1.0,
+                        ],
+                        "_top_k_voting": 1,
+                        "_min_voting_prob": 0.2,
+                        "_min_prob_differential": 0.2,
+                        "_top_k_labels": 3,
+                        "_min_top_label_prob": 0.35,
+                        "_DataLabelerColumn__calculations": {},
+                    },
+                }
+            )
+
+            self.assertEqual(serialized, expected)
```

Comment:
I think our standards are `expected` first.

Reply:
eh not really a standard... we've done `serialized, expected` in `test_base_column_profilers.py`, `test_categorical`, `test_datetime_column_profile`, `test_int_column` , and `test_numeric`, `test_order`

Reply:
> eh not really a standard... we've done `serialized, expected` in `test_base_column_profilers.py`, `test_categorical`, `test_datetime_column_profile`, `test_int_column` , and `test_numeric`, `test_order`

Specifically in this test? IIRC , `self.assert...` returns a response where it assumes the first var is the expected.

Reply:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertEqual